src/sql/base/browser/ui/table/tableCellEditorFactory.ts

```diff
@@ -174,4 +175,76 @@ export class TableCellEditorFactory {
 		}
 		return TextEditor;
 	}
+
+	public getDropDownEditorClass(context: any, defaultOptions: string[]): any {
```

Comment:
These classes seem to be of type Slick.Editor (or even Slick.Text since it seems to be specifically a text editor...) - could you use that instead of any? And have the class definition extend that?